### PR TITLE
Update charter to include collaboration with Immersive Web Working Group

### PR DESCRIPTION
**Pull request by skonig-google**
Propose that this group also coordinate with the Immersive Web Working Group.

### CODE REVIEW

**Comment by anssiko** [comment]
Thanks for the suggestion! Immersive Web and Machine Learning should certainly coordinate.

I'm just wondering whether the Immersive Web _Community Group_ would be a more appropriate coordination point?

The [IWWG scope](https://www.w3.org/2018/09/immersive-web-wg-charter#scope) does not explicitly mention object detection & machine learning (it does state "receive updated information about the device's environment", but that's not very specific). OTOH, the IWCG is scoped to incubate new features (see the [IWCG proposals repo](https://github.com/immersive-web/proposals)) which looks like a better fit.

WDYT @skonig-google? If you agree, could you update the PR accordingly. Thanks!

Looping in @cwilso for comments.

**Comment by skonig-google** [comment]
I think it would make sense to interface with both.  It's certainly true that object detection for example isn't part of the initial scope of work for the WG, but it's definitely on the long-term roadmap and is being incubated in the CG.  Once we're ready for it, I expect the WG to pick it up.  I'll update the PR accordingly.

**Comment by cwilso** [comment]
I agree it makes sense to coordinate with both - in practice, the overlap is nearly complete, and we coordinate quite heavily between the two anyway.

**Comment by skonig-google** [comment]
Updated, PTAL.

File: index.html [hunk 1]
```diff
@@ -262,6 +262,19 @@ <h2 id="coordination">
         Learning use cases.
       </dd>
     </dl>
+    <dl>
+      <dt>
+        Immersive Web <a href="https://www.w3.org/immersive-web/">Working Group</a> and <a href="https://www.w3.org/community/immersive-web/">Community Group</a>
+      </dt>
+      <dd>
+        The Immersive Web Working Group and Community Groups aim to bring high-performance
```
Comment (anssiko) [inline index.html]: Community Group

(Singular)

Comment (skonig-google) [reply index.html]: Changed, thanks.

**Pull request merged**